tox.ini

```diff
@@ -7,6 +7,7 @@ deps =
     isort
     black == 21.9b0
     yamllint
+    click == 8.0.4
```

Comment:
Could you create a separate PR of this change?

Reply:
Done 
github.com/red-hat-storage/ceph-qe-scripts/pull/250
Will update this PR